package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lib-oauth-tooling",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
Nevertheless I would like to test the version in the wild before publishing it to make sure nothing is broken.